rubygem-hammer_cli_foreman_discovery/foreman_discovery.yml

```diff
@@ -0,0 +1,2 @@
+:foreman_discovery:
```

Comment:
Please also submit this to the core repo, I'd like to be able to delete it in the next release.